MapboxSearch/gradlew

```diff
@@ -169,4 +169,5 @@ if [ "$(uname)" = "Darwin" ] && [ "$HOME" = "$PWD" ]; then
   cd "$(dirname "$0")"
 fi
 
+echo "$JAVACMD" "$@"
```

Comment:
Why do you need this change?  This file shouldn't be updated manually

Reply:
This was to figure out the exact version of Java that we have in CI/CD. Rolled back now.